Source/Game/Game/Rendering/Model/ModelLoader.cpp

```diff
@@ -32,7 +32,7 @@ static const fs::path complexModelPath = dataPath / "ComplexModel/";
 
 ModelLoader::ModelLoader(ModelRenderer* modelRenderer)
 	: _modelRenderer(modelRenderer)
```

Comment:
Not sure what happened in this file, but looks like difference between tabs & spaces